packages/api/app/Utils/slugify.js

```diff
@@ -1,8 +1,19 @@
 const slugify = require('slugify');
 
-const Hash = use('Hash');
 const Database = use('Database');
 
+const removeAccents = (text) => {
+	let textWithoutAccents;
+	textWithoutAccents = text.toLowerCase();
+	textWithoutAccents = textWithoutAccents.replace(new RegExp('[ÁÀÂÃ]', 'gi'), 'a');
+	textWithoutAccents = textWithoutAccents.replace(new RegExp('[ÉÈÊ]', 'gi'), 'e');
+	textWithoutAccents = textWithoutAccents.replace(new RegExp('[ÍÌÎ]', 'gi'), 'i');
+	textWithoutAccents = textWithoutAccents.replace(new RegExp('[ÓÒÔÕ]', 'gi'), 'o');
+	textWithoutAccents = textWithoutAccents.replace(new RegExp('[ÚÙÛ]', 'gi'), 'u');
+	textWithoutAccents = textWithoutAccents.replace(new RegExp('[Ç]', 'gi'), 'c');
+	return textWithoutAccents;
+};
+
```

Comment:
Esses métodos retornam uma nova string, então, vc pode encadeá-los.

```suggestion
const removeAccents = (text) =>
	text
		.toLowerCase()
		.replace(new RegExp('[ÁÀÂÃ]', 'gi'), 'a')
		.replace(new RegExp('[ÉÈÊ]', 'gi'), 'e')
		.replace(new RegExp('[ÍÌÎ]', 'gi'), 'i')
		.replace(new RegExp('[ÓÒÔÕ]', 'gi'), 'o')
		.replace(new RegExp('[ÚÙÛ]', 'gi'), 'u')
		.replace(new RegExp('[Ç]', 'gi'), 'c');
```